keyboards/nightly_boards/octopad/rules.mk

```
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes         # Mouse keys
+EXTRAKEY_ENABLE = yes         # Audio control and System control
+CONSOLE_ENABLE = no           # Console for debug
+COMMAND_ENABLE = no           # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no         # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no              # USB Nkey Rollover
+BACKLIGHT_ENABLE = no         # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes         # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no         # Enable Bluetooth
+AUDIO_ENABLE = yes             # Audio output
+ENCODER_ENABLE = yes          # Enable Rotary Encoders
+
+# Added encoder Action
+SRC +=  ../../encoder_action.c
```

Should only need to be: 
```suggestion
SRC += encoder_action.c
```